apps/vaults/hooks/useSolverPortals.ts

```diff
@@ -56,7 +56,7 @@ function usePortalsQuote(): [
 
 		if (canExecuteFetch) {
 			try {
-				return axios.get(`https://api.portals.fi/v1/portal/${NETWORK.get(1)}/estimate`, {params});
+				return (await axios.get(`https://api.portals.fi/v1/portal/${NETWORK.get(1)}/estimate`, {params})).data;
```

Comment:
Let's go with the same pattern as the others
```suggestion
				const {data: estimate} = await axios.get(`https://api.portals.fi/v1/portal/${NETWORK.get(1)}/estimate`, {params});
				return estimate;
```